### PR TITLE
Update to mongo-rust-driver 2.x

### PR DESCRIPTION
**Pull request by Samuel-B-D**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.8.0] 2021-12-09
+
+### Changed
+
+- **BREAKING:** MongoDB driver and Tokio update
+  tokio 1.x
+  mongodb 2.0
```
Comment (CBenoit) [inline CHANGELOG.md]:
```suggestion
- **BREAKING:** Update `mongodb` to `2.x`
```

`tokio` is a transitive dependency, but we don't directly depend on it. I think it's enough to specify the `mongodb` update for the breaking change.

File: Cargo.toml [hunk 2]
```diff
@@ -12,16 +12,18 @@ readme = "README.md"
 include = ["src/**/*", "README.md", "CHANGELOG.md"]
 
 [dependencies]
-mongodb = { version = "^1.2", default-features = false }
+mongodb = { version = "2.0.2", default-features = false }
```
Comment (CBenoit) [inline Cargo.toml]:
```suggestion
mongodb = { version = "2", default-features = false }
```

"2" for `>=2.0.0 && <3.0.0`

**Review by CBenoit — CHANGES_REQUESTED** [review]
Nice, thank you! Just two suggestions related to the CHANGELOG and `mongodb` dependency version specification.

File: CHANGELOG.md (hunk 1 above)
Comment (Samuel-B-D) [reply CHANGELOG.md]: Gotcha

**Review by CBenoit — APPROVED** [review]
LGTM :tada:

**Pull request merged**